src/org/traccar/protocol/Tk103Protocol.java

```diff
@@ -40,7 +40,22 @@ public Tk103Protocol() {
                 Command.TYPE_REBOOT_DEVICE,
                 Command.TYPE_SET_ODOMETER,
                 Command.TYPE_ENGINE_STOP,
-                Command.TYPE_ENGINE_RESUME);
+                Command.TYPE_ENGINE_RESUME,
+                //T580W commands:
+                "T580W_positionSingle",
```

Comment:
You need to map commands to standard ones. Applies to all.

Reply:
1. Decide which command to use based on some configuration attribute. See GT06 encoder for example.
2. Either remove or add it to the command types if you think it's generic enough that would be useful for many protocols. What does `multiControllerOn` do?

Reply:
from user guide from T580W: https://goo.gl/M4ancp: 
`12. Multi Controller
Allows any cellphone number to check the device location and status besides the
master number.`

Reply:
This might be too specific. I would recommend to remove it for now and leave only generic enough commands.

Reply:
done

Reply:
> ...See GT06 encoder for example....
How to set this configuration attribute from test case code, so i can test my implementation of the command? Thanks!

Reply:
Add a setter.